### ISSUE TITLE
Weird binding problem in scalajs!

### ISSUE THREAD

**Issue by scalway** [outr/reactify#26]
sometimes binding.detach() does not work.

example:
https://scalafiddle.io/sf/BR0hHAZ/2

**Comment by darkfrog26** [comment]
I'm unable to determine what is not working right.  I created a test case:
https://github.com/outr/reactify/blob/master/shared/src/test/scala/test/BindingSpec.scala#L87

Everything appears to be working fine.

**Comment by scalway** [comment]
yes... i've messed up this scalafiddle. It was not clear what happend. Here is simpler example.
https://scalafiddle.io/sf/ppKwo7c/2

**Comment by darkfrog26** [comment]
Ah, I see.  I'll try to investigate further when I have a few minutes.  It looks to be a browser issue.  I would bet it has something to do with object equality since it doesn't seem to happen in NodeJS or JVM.

**Comment by scalway** [comment]
It seams to be much more subtle than i though. Thanks.
